framework/test/unit/engine/legacy/legacy_chain_handler.spec.ts

```diff
@@ -48,7 +122,17 @@ describe('Legacy Chain Handler', () => {
 		});
 	});
 
-	describe('syncBlocks', () => {
-		it.todo('test behaviors');
+	describe('sync', () => {
+		it('test behaviors', async () => {
```

Comment:
I've noticed you've not used the convention `it(should...)` as well as you've not defined the behavior that is being tested.